### ISSUE TITLE
Run status code tests with remote client

### ISSUE THREAD

**Issue by pakrym** [aspnet/IISIntegration#987]
Custom error page module behaviour differs based on if the client is remote and it might affect response body.

See https://github.com/aspnet/IISIntegration/issues/978 https://github.com/aspnet/IISIntegration/issues/983 and https://github.com/aspnet/IISIntegration/pull/985

**Comment by jkotalik** [comment]
Docker could be a solution for this. It depends on how much we care about testing this behavior on win7.

**Comment by mikeharder** [comment]
@jkotalik : Yes, containers make testing this fairly easy, with the caveat we can only cover a limited number of OS.  I've already verified that requests to containers are considered "remote".

**Comment by shirhatti** [comment]
We've manually verified this, but we do need automated coverage
